transport_nantes/asso_tn/utils.py

```diff
@@ -96,8 +96,18 @@ def token_valid(encoded_timed_token, test_value_now=None):
     """
     try:
         timed_token = urlsafe_base64_decode(encoded_timed_token).decode()
-    except (TypeError, ValueError):
-        logger.info(f"token_valid: invalid token ({encoded_timed_token})")
+    except (TypeError, ValueError) as e:
+        # On the 31/05/2022 :
+        # While sending a press release, it appears that at least 10 token out
+        # of 24 were invalid. We know this because we got 10 times a
+        # "token_valid: invalid token" error on GET requests to the
+        # 'mailing_list:press_subscription_management' url, seconds after the
+        # send.
```

Comment:
Good.  You might note all the things you learned yesterday whilst debugging this.  For example, that the error occurred in `token_valid()`, whatever else you learned.

You spent (if I understood correctly) over an hour on this.  Surely you learned more than you can summarise in two sentences!

Some of that summary can go in the issue, of course.  The code comment needs to get future devs up to speed on what the issue is, the issue can provide all the gory detail.

Reply:
Reworded + added document to issue